### DISCUSSION TITLE
2.34.0

### DISCUSSION THREAD

**Discussion by TheLastProject** [CatimaLoyalty/Android#2219] [Releases]
- Add Passbook (.pkpass) support
- Fix import of transparent PDF files
- Improve display of transparent thumbnails

<hr /><em>This discussion was created from the release <a href='https://github.com/CatimaLoyalty/Android/releases/tag/v2.34.0'>2.34.0</a>.</em>

**Comment by VasilisKos** [comment]
Thank you so much for this long waited feature! Tried it with a sample pkpass file and looks like it's working great. There's only one thing missing. The .pkpass file type is not associated with Catima, and Android doesn't offer it as an option when trying to open such files through a file browser (using stock Android 14). 
Can you please do something about that?

**Reply by TheLastProject** [reply]
Hmm, I can't reproduce that. I'm on Android 14 too, using "com.android.documentsui" as File Manager (which I believe is the built-in one). I don't know how to debug these things but just as a guess, does your file manager know it's an "application/vnd.apple.pkpass"? If you view the file info in the file manager, what does it show as mimetype?

**Reply by VasilisKos** [reply]
I'm using Solid Explorer. In the file info it says "application/*".

**Reply by TheLastProject** [reply]
That sounds like Solid Browser doesn't correctly understand pkpass files, the mime type of a pkpass file should be `application/vnd.apple.pkpass`. I've created #2220 to create a workaround, but I think you should try contacting the Solid Explorer developer and ask them to look into this, I'm pretty sure Catima is doing it right :)

**Reply by VasilisKos** [reply]
I've sent a mail to the dev, but I'm not sure if he's going to do anything about it. The app has millions of downloads, and he's probably bombed with mails every day. I was out of luck with previous mails I've sent him... The support sucks, though I've bought the Pro version...🤨
Let's hope for the best...

**Reply by VasilisKos** [reply]
Just got a mail from the dev, saying that he'll fix it in the next update. Let's see.

**Reply by VasilisKos** [reply]
Today's Solid Explorer update fixed the issue, as promised. ☺️

**Reply by TheLastProject** [reply]
Nice :) Thanks for keeping us updated!
